datamodel/src/main/java/com/hartwig/serve/datamodel/serialization/util/ActionableFileUtil.java

```diff
@@ -36,11 +41,12 @@ public static String header() {
         return new StringJoiner(FIELD_DELIMITER).add("source")
                 .add("sourceEvent")
                 .add("sourceUrls")
-                .add("studyNctId")
-                .add("studyTitle")
-                .add("studyAcronym")
-                .add("studyGender")
-                .add("countriesOfStudy")
+                .add("nctId")
+                .add("title")
+                .add("acronym")
+                .add("genderCriterium")
+                .add("countries")
```

Comment:
Is this field really just countries or actually "countries and cities"? Code seems to imply both